AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -322,6 +323,14 @@ public static boolean hasReceivedDiffs() {
     return hasReceivedDiffs;
   }
 
+  public static Map<String, Object> getVariantDebugInfo() {
+    return variantDebugInfo;
+  }
+
+  public static void setVariantDebugInfo(Map<String, Object> _variantDebugInfo) {
```

Comment:
style: no "_" in var name

I think it's better to mark this as `@Nullable`, and set this.variantDebugInfo to empty map.